### ISSUE TITLE
Trying to rename an aura doesn't work reliably.

### ISSUE THREAD

**Issue by Nogamara** [WeakAuras/WeakAuras2#2813]
## Describe the bug

Trying to rename an aura that was just duplicated doesn't work reliably.

**Do you have an error log of what happened?**
 
nothing

### To Reproduce

  * First, duplicate an aura, the name field on the left is in edit mode.

What doesn't work:

  * change text, hit enter - nothing happens or instantly reverts
  * right-click, rename, editbox appears, write something, press enter - nothing happens or instantly reverts
  * Information tab -> name, change it, click okay - instantly reverts
  * Information tab -> name, change it, press enter - instantly reverts

What seems to work most of the time:

  * Hit Escape immediately, it's named "originalname 2", then go to information tab and rename it

**Which version of WeakAuras are you using?**

3.1.4

**Are you on World of Warcraft Classic or Retail?**

- [ ] Classic
- [x] Retail

**Was it working in a previous version? If yes, which was the last good one?**

This is the first time I use WeakAuras

**Comment by InfusOnWoW** [comment]
I can't reproduce.

Do note that all aura names must be unique.

**Comment by Nogamara** [comment]
> Do note that all aura names must be unique.

Thanks, that was a good hint.

I downloaded https://wago.io/Nyga0qXLz/10 and it has some Sub-Auras named after Warrior skills, e.g. Recklessness and Siegebreaker.

But I made my own group with subauras, but I managed to create a 2nd one called Recklessness but with a space at the end...

So I guess an error message like "that name is already taken" would be super helpful as it's not messaged at all.
